src/Infrastructure/Services/FraudProtectionService.cs

```diff
@@ -82,6 +84,28 @@ public FraudProtectionService(IOptions<FraudProtectionSettings> settings, IToken
             return response;
         }
 
+        private async Task<HttpResponseMessage> PostAsyncAP<T>(
```

Comment:
Not a huge fan of another nearly duplicate method besides, I think, setting the metadata.  I think it's fine for now though, especially considering the non-AP one may go away sometime this year.

Reply:
got rid of the duplicate method.  PostAsync will now check the type of the content it is trying to send before adding metadata.